frontend/app/views/spree/address/_form.html.erb

```diff
@@ -65,30 +67,31 @@
             class: !have_states ? 'required' : '',
             style: have_states ? 'display: none;' : '',
             disabled: have_states,
-            autocomplete: address_type + ' address-level1'
+            autocomplete: address_type + ' address-level1',
+            required: true
```

Comment:
I don't think the `!have_states? 'required'` logic makes sense for anything else besides styles here. It's simple enough to remove the `required: true` attribute for a fringe case where `address-level1` would not be required, but the form is for Debit/Credit or Shipping which **do** require an address.